modules/generator/processor/servicegraphs/servicegraphs.go

```diff
@@ -270,7 +270,7 @@ func (p *Processor) upsertPeerNode(e *store.Edge, spanAttr []*v1_common.KeyValue
 }
 
 // upsertDatabaseRequest handles the logic of adding a database edge on the
-// graph.  If we have a db.name or db.system attribute, we assume this is a
+// graph.  If we have a db.name or db.system or db.system.name attribute, we assume this is a
```

Comment:
We will want to update the documentation for this change here `docs/sources/tempo/metrics-generator/service_graphs/_index.md`.